### PR TITLE
Create 2022-02-02-Wormhole.md

### PR DESCRIPTION
**Pull request by Diam0ndAce**
Adding an article about Wormhole hack in 2022

### CODE REVIEW

**Review by eric-oconnor — APPROVED** [review]
Good article, it aligns will with the information here: https://www.theverge.com/2022/2/3/22916111/wormhole-hack-github-error-325-million-theft-ethereum-solana

This is good as it gives us a place to reference the facts and assure accuracy of the data presented.

File: content/attacks/posts/2022-02-02-Wormhole.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## Timeline
+
+- **February 2, 2022:** Wormhole upload on their Github disguised security fix. Most likely, hacker notice that and start planning the attack. Just hour later the hacker exploit vulnerability in the Wormhole's bridge protocol and stole  $325 million.
+- **February 3, 2022:**  Token bridge is back up and Jump Crypto stepped in to replace the stolen funds.
+
+## Security Failure Causes
+
+- **Weakness in bridge protocol:** Wormhole bridge protocol had a vulnerability on the Solana side that allowed attacker mint wETH.
+- **Lack of the security awareness:** Wormhole team know about weakness in their bridge and the necessary code to fix this vulnerability had been written. But the code was uploaded to the Github repository before it was applied to the production app. This fact allows a hacker to successfully exploit the protocol vulnerability.
```
Comment (jhirschkorn) [inline content/attacks/posts/2022-02-02-Wormhole.md:28]: The Wormhole team knew about weaknesses in their bridge but the necessary code to fix it had been written.

This fact allowed...

File: content/attacks/posts/2022-02-02-Wormhole.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+date: 2022-02-02
+custodians: Wormhole
+categories: Custodian
+title: Wormhole lost $325 million due to hacker attack
```
Comment (jhirschkorn) [inline content/attacks/posts/2022-02-02-Wormhole.md:5]: add quotes to the title

**Review by jhirschkorn — CHANGES_REQUESTED** [review]

**Comment by jhirschkorn** [comment]
Conflict. Another article on this topic already exists.

**Pull request closed without merging**